pkgs/tools/security/yubihsm-shell/default.nix

```diff
@@ -0,0 +1,42 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, openssl
+, libusb
+, libedit
+, curl
+, gengetopt
+, pkg-config
+, pcsclite
+, help2man
+}:
+
+stdenv.mkDerivation rec {
+  pname = "yubihsm-shell";
+  version = "2.3.1";
+
+  src = fetchFromGitHub {
+    owner = "Yubico";
+    repo = "yubihsm-shell";
+    rev = version;
+    sha256 = "sha256-D0kXiwc+i6mKA4oHuHjgXUmLMsmY5o/VI+1aCWtNjC0=";
+  };
+
+  nativeBuildInputs = [ pkg-config cmake help2man ];
+
+  buildInputs = [ libusb libedit curl gengetopt pcsclite openssl ];
+
+  # help2man fails due to their cmake configuration
+  cmakeFlags = [
+    "-DWITHOUT_MANPAGES=1"
+  ];
+
+  meta = with lib; {
+    description = "yubihsm-shell and libyubihsm";
+    homepage = "https://github.com/Yubico/yubihsm-shell";
+    maintainers = with maintainers; [ matthewcroughan ];
+    license = licenses.apsl20;
+    platforms = platforms.unix;
```

Comment:
```suggestion
```

I'd just remove this and leave the buildGoModules default pulled from go

I only put platforms when it's more restrictive than that

Might be worth testing this on mac? or just putting platform as linux?
Or we could leave it as is until a mac user raises an issue?